test/factories/tags.rb

```diff
@@ -0,0 +1,15 @@
+# == Schema Information
+#
+# Table name: tags
+#
+#  id         :integer          not null, primary key
+#  name       :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+FactoryGirl.define do
+  factory :tag do
+    name { "name" + ('a'..'z').to_a.sample(26).join }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.